source/ns_cmdline.c

```diff
@@ -54,10 +54,15 @@
 //#define TRACE_DEEP
 //#define TRACE_PRINTF
 
-
 #ifdef TRACE_PRINTF
 #undef tr_debug
 #define tr_debug(...) printf( __VA_ARGS__);printf("\r\n")
+#else
```

Comment:
@jupe : From my understanding of code in ns_cmdline.c it mbed_trace functionality is not used, it’s all disabled by the use of exclude filter and I don’t see it as enabled anywhere. I see trace prints only when flag is enabled for internal tracing, else it is disabled for the module. 

Please let me know which all tests I can run to verify this PR.

Reply:
> it’s all disabled by the use of exclude filter

Correct, but it can be enabled from outside the library by simply disabling the filter after the library has been initialized.

These new defines need to be placed under a different flag to allow enabling the traces without enabling the above tr_debug macro.

Reply:
Those macros also need to be undeffed before defining them to avoid multiple definition.